lib/generators/apostle/install_generator.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails/generators'
```

Comment:
You've got this file in both `apostle` and `apostle_rails`. Given that the Gem and generators will only work with Rails, it's probably best to have this under `apostle` and with a module of `Apostle` rather than `ApostleRails`. So if you moved the `initializer.rb` into `lib/generators/apostle/` and removed `lib/generators/apostle_rails`, and updated the module to `Apostle` then we'd be good to go!